tools/darwin/packaging/darwin_embedded/mkdeb-darwin_embedded.sh.in

```diff
@@ -75,7 +75,11 @@ echo "Conflicts: $PACKAGE"                      >> $DIRNAME/$PACKAGE/DEBIAN/cont
 echo "Replaces: $PACKAGE"                       >> $DIRNAME/$PACKAGE/DEBIAN/control
 echo "Priority: Extra"                            >> $DIRNAME/$PACKAGE/DEBIAN/control
 echo "Version: $VERSION-$REVISION"                >> $DIRNAME/$PACKAGE/DEBIAN/control
-echo "Architecture: @CMAKE_SYSTEM_NAME@-@CPU@"                 >> $DIRNAME/$PACKAGE/DEBIAN/control
+if [ "@PLATFORM@" == "iphoneos" ]; then
+  echo "Architecture: iphoneos-arm"                 >> $DIRNAME/$PACKAGE/DEBIAN/control
+else
+  echo "Architecture: appletvos-arm64"                 >> $DIRNAME/$PACKAGE/DEBIAN/control
```

Comment:
According to the snippet in your PR description, shouldn't this be darwin-arm64?

Reply:
Good question... I am still not sure between Darwin and appletvos.
According to Memphiz yab branch code it should be appletvos, but the user error shows darwin.
Anybody here have a jailbroken Apple TV to try both?

Reply:
Does @PLATFORM@ could the correct platform name?

Reply:
Yes, but we also need to use `arm` for iOS and `arm64` for tvOS :-/
What do you mean by "the deb names"?

Reply:
Example filename ms of the generated .deb files

Reply:
And `iphoneos-arm64` won’t work? Pity

Reply:
I don't is there is any convention, on Cydia, for example, the NewTerm2 app us this filename: `ws.hang.newterm2_2.4_iphoneos-arm.deb`. I can look for others examples if you want.

I will try tomorrow with `iPhones-arm64`.

Reply:
So, results of my test on iPhone 7:
* `iphoneos-arm`: `dpkg` is happy and install Kodi correctly
* `iphoneos-arm64`: `dpkg` error (package architecture (xxxxx) does not match system (iphoneos-arm))
* `darwin-arm`: idem
* `darwin-arm64`: idem

Reply:
What does `idem` mean?

Reply:
Oh sorry, I thought it was international. "Idem" means "same result as above".

Reply:
Nope, well not that I’ve heard 😉

Reply:
It must be French then 😋